src/libs/Database.ts

```diff
@@ -11,6 +11,7 @@ export default class Database {
                 )
                 .then(() => {
                     console.log('Successfully Connected to Db');
+                    seed();
```

Comment:
here we are seeding to the database, right?

Reply:
Yes